optuna/trial/_frozen.py

```diff
@@ -166,29 +170,72 @@ def _validate(self):
                 )
 
     @property
-    def distributions(self):
-        # type: () -> Dict[str, BaseDistribution]
+    def number(self) -> int:
+        return self._number
```

Comment:
In the Optuna coding style, we should insert a blank line as follows. Could you insert blank lines to other places?
```suggestion
    def number(self) -> int:
   
        return self._number
```